lib/gemini.js

```diff
@@ -141,6 +143,16 @@ module.exports = class Gemini extends QEmitter {
         return this.config.forBrowser(browserId).desiredCapabilities;
     };
 
+    _validateOptions(options) {
+        if (typeof options === 'string') {
+            console.warn(chalk.yellow('You have to pass an object with options instead of string'));
```

Comment:
вот использую я gemini через API. Запускаю свою тулзу, и мне в консоль вываливается:
```
Тут какие-то мои логи
...
You have to pass an object with options instead of string
```
и я такой "а где, бл@?!"